osm2lanes/src/locale.rs

```diff
@@ -22,9 +23,16 @@ impl Locale {
 
     #[must_use]
     #[allow(clippy::unused_self)]
-    pub fn travel_width(&self, designated: &Designated) -> Metre {
+    pub fn travel_width(&self, designated: &Designated, _highway: HighwayType) -> Metre {
         match designated {
-            Designated::Motor | Designated::Bus => Metre::new(3.5),
+            Designated::Motor | Designated::Bus => {
+                let uk = Country::the_united_kingdom_of_great_britain_and_northern_ireland();
+                match &self.country {
```

Comment:
Is it worth documenting where these values were found? If they're just reasonable guesses based on eyeballing things, also fine

Reply:
yes, I'll add documentation.